docs/possible_stardis_outputs/possible_stardis_outputs.ipynb

```diff
@@ -0,0 +1,746 @@
+{
```

Comment:
<p>Careful about your imports. Not sure you use download_atom_data anywhere in this notebook. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/stardis/pull/230/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Addressed in '[stellar_radiation and import fix](https://github.com/tardis-sn/stardis/pull/230/commits/da2fff3e2f6b866af6ebeff0ef9e3499922d21e5)'